### ISSUE TITLE
Sign plugin

### ISSUE THREAD

**Issue by julienmalik** [senbox-org/snap-rut#1]
The plugin is not signed, so it emits a warning when installing it from the update center

**Comment by marpet** [comment]
Yes, this will be fixed when it will be integrated in the toolbox. But thanks for the notification.
